src/events.ts

```diff
@@ -56,15 +56,15 @@ function unbindable(el: EventTarget, eventName: string, callback: MouchBind, opt
 
 function startDragOrDraw(s: State): MouchBind {
   return e => {
-    if (isRightButton(e) && s.draggable.current) drag.cancel(s);
-    else if ((e.shiftKey || isRightButton(e)) && s.drawable.enabled) draw.start(s, e);
+    if (isRightClick(e) && s.draggable.current) drag.cancel(s);
```

Comment:
lichess supports disabling auto-promotion by moving the pawn while holding `ctrl`. But now it will cancel the drag. Better keep `isRightButton` here.